.github/workflows/ci.yml

```diff
@@ -1,15 +1,12 @@
-name: Tests
```

Comment:
I think we can have also the tests job, is there an issue if we keep both? We can have multiple jobs, but definetely we can set prerequisite to have the tests pass before publishing into VS Marketplace.

Reply:
They were removed by mistake, I've added them back